### PR TITLE
Fixes #1347 - single quotes to double quotes for strings

### LINKED ISSUE #1347: eslint: change to double quotes for strings in JS
Single quotes in JS feels gross, I think I chose that for consistency with our Python style, but I don't like it anymore.

Unless somebody objects, I'm going to do submit a PR to do this in the next few days.

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr 

Please also test the grunt tasks / new npm scripts. Just to be sure I didn't break anything.

### CODE REVIEW

**Comment by miketaylr** [comment]
Awesome -- thank you!

File: package.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch" : "grunt watch",
     "build": "grunt",
+    "lint": "grunt eslint",
```
Comment (miketaylr) [inline package.json]: Can you file a follow up bug: our grunt eslint task should be checking all .js files and currently doesn't (my bad, probably). It looks like what you use for the fix command is more what we want.

Comment (miketaylr) [inline package.json]: @zoepage ah, if we document eslint anywhere in CONTRIBUTING.md, it might be good to add these commands as alternatives.

**Comment by magsout** [comment]
@zoepage ❤👍

**Pull request merged**